enumerated_constants.gemspec

```diff
@@ -16,8 +16,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activesupport"
-
+  spec.add_development_dependency "activesupport"
```

Comment:
If we swap `expect(x).to be_in(y)` with `expect(y).to include(x)`, we don't need this for development either.